test/message_parser_test.dart

```diff
@@ -0,0 +1,116 @@
+// @dart=2.10
```

Comment:
Please add a copyright statement (you can see this in the other files; for new files we'd update to have the current year).